fastapi/app/main.py

```
@@ -119,117 +56,154 @@ async def process_video(file: UploadFile = File(...)):
+                except:
+                    pass
+
+def get_video_duration(path: str) -> float:
+    try:
+        result = subprocess.run(
+            ["ffprobe", "-v", "error", "-show_entries", "format=duration",
+             "-of", "default=noprint_wrappers=1:nokey=1", path],
+            stdout=subprocess.PIPE, text=True
+        )
+        return float(result.stdout.strip())
+    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)